llvm/test/CodeGen/RISCV/cheri/float-mem.ll

```diff
@@ -0,0 +1,114 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; Check that we can load/store single-precision floating point values via a
+; capability pointer without capability mode.
+; RUN: %riscv32_cheri_llc -mattr=-f -verify-machineinstrs < %s \
+; RUN:   | FileCheck -check-prefix=RV32IXCHERI %s
```

Comment:
I'd like these to be explicit about being hybrid. Generally I've gone with ARCH-ABI for the CHECK lines as that seems to be what RISC-V does, so RV32IXCHERI-ILP32, RV32IFXCHERI-ILP32F etc, and similarly for what are currently PURECAP-*.

Reply:
Sounds good, will update tomorrow morning.